mantidproject/mantid
# Stitch1DMany new property 'ScaleFactorFromPeriod'

When stitching workspaces from multiple groups together, the algorithm currently allows a global scale factor to be specified using properties `UseManualScaleFactor` and `ManualScaleFactor`. This new property `ScaleFactorFromPeriod` allows a set of global scale factors to be determined from workspaces in groups from a specific period and applies them to workspaces in all other periods.

This property is only used when `UseManualScaleFactor` is true and `ManualScaleFactor` is set to its default value. Furthermore, the documentation regarding the `Stitch1DMany` algorithm has been updated and improved.

**To test:**

- Ensure all unit tests pass.
- Ensure that the `ScaleFactorFromPeriod` property accomplishes what it was meant to do.
- Ensure that the documentation is up to date and correct.

Fixes #17819.

*Release notes have been updated.*

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
